packages/validator/src/util/externalSignerClient.ts

```diff
@@ -157,20 +164,34 @@ export async function externalSignerPostSignature(
 export async function externalSignerUpCheck(remoteUrl: string): Promise<boolean> {
   const res = await fetch(`${remoteUrl}/upcheck`, {
     method: "GET",
-    headers: {"Content-Type": "application/json"},
+    headers: {Accept: MediaType.json},
   });
 
-  const data = await handlerExternalSignerResponse<{status: string}>(res);
+  const data = await handleExternalSignerResponse<{status: string}>(res);
   return data.status === "OK";
 }
 
-async function handlerExternalSignerResponse<T>(res: Response): Promise<T> {
+async function handleExternalSignerResponse<T>(res: Response): Promise<T> {
   if (!res.ok) {
     const errBody = await res.text();
-    throw Error(`${errBody}`);
+    throw Error(errBody || res.statusText);
+  }
+
+  const contentType = res.headers.get("content-type");
+  if (contentType === null) {
```

Comment:
This is a breaking change from previous behavior. Would it be worth defaulting to JSON handling if no `content-type` is set? We could show a warning then.

Reply:
Both Diva and Web3signer are setting this header and I would prefer to be more strict about this